src/components/browse/components/FacetList/ExtendedDescriptionPopoverIcon.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+import React, { useMemo, useState, useRef } from 'react';
+import memoize from 'memoize-one';
+import PropTypes from 'prop-types';
+import _ from 'underscore';
+import Fade from 'react-bootstrap/esm/Fade';
+
+import Popover from 'react-bootstrap/esm/Popover';
+
+
+export function FacetListPopoverController (props) {
+    const { children } = props;
+    // In this state we will store the entirety of the Popover JSX to display
+    // to allow flexibility in what to display in Popover contents
+    // in case we add more types of Popovers later.
+    const [ openPopoverID, setOpenPopoverID ] = useState(null);
+    const adjustedChildren = React.Children.map(children, function(child){
+        return React.cloneElement(child, { openPopoverID, setOpenPopoverID });
+    });
+    return (
+        <React.Fragment>
+            { adjustedChildren }
+
+        </React.Fragment>
+    );
+}
+
+
+/**
+ * @todo Potentially refactor to use ReactDOM.createPortal along w. semi-opaque dark background that will close this onClick.
```

Comment:
Random question: is ReactDOM.createPortal actually in use anywhere on CGAP currently? I remember I was looking around the repo for an example a few weeks back when I was messing around with creating an interpretation/annotation overlay. It looked like there was a lot of infrastructure to support the use of it, but that it wasn't actually being used yet.

Reply:
It's being used for [BigDropdownMenu and the 'actions' slide out pane on ItemViews](https://github.com/dbmi-bgm/cgap-portal/search?q=ReactDOM.createPortal) currently.